doc/source/examples_source/00-basic/07-volume_render.py

```diff
@@ -0,0 +1,199 @@
+"""
+.. _volume_render:
+
+Volume Rendering to Visualize Flow
+=======================================================
+
+Utilze EnSight to demonstrate the value of Volume Rendering
+as a tool to visulize flow in 3D
+
+Intended to work with EnSight version 24.2 or later
+
+"""
+
+###############################################################################
+# Start an EnSight session
+# ------------------------
+# Launch and connect to an instance of EnSight.
+# This example uses a local EnSight installation.
+from ansys.pyensight.core import LocalLauncher
+
+# batch is default (no visible ensight session)
+session = LocalLauncher().start()
+# use below for interactive debugging with a visible ensight session 
+#session = LocalLauncher(batch=False).start()
+
+# Setup shortcuts for long winded calls.
+sesse = session.ensight
+eocore = sesse.objs.core
+eoutil = sesse.utils
+
+###############################################################################
+# Load a dataset
+# --------------
+# Load Shuttle Session file included in the EnSight installation and render
+#
+# .. image:: /_static/07_volume_render_0.png
+
+###############################################################################
+# Load a dataset
+# --------------
+# Load Shuttle data included in the EnSight installation and render
+#
+# .. image:: /_static/03_ptrace_0.png
+
+xyz_file = f"{session.cei_home}/ensight{session.cei_suffix}/data/plot3d/shuttle.xyz"
+q_file = f"{session.cei_home}/ensight{session.cei_suffix}/data/plot3d/shuttle.q"
+session.load_data(
+    data_file=xyz_file,
+    result_file=q_file,
+    file_format="PLOT3D",
+    representation="3D_feature_2D_full",
+)
+session.show("image", width=800, height=600)
+
+###############################################################################
+# The PLOT3D reader only reads the volume by default. Now, extract a
+#  particular IJK range for the surface of the shuttle
+# ------------------------------------------------------------------
+
+sesse.data_partbuild.begin()
+sesse.case.select("Case 1")
+sesse.data_partbuild.data_type("structured")
+sesse.data_partbuild.group("OFF")
+sesse.data_partbuild.select_begin(1)
+sesse.data_partbuild.domain("all")
+sesse.data_partbuild.noderange_i(1, 53)
+sesse.data_partbuild.noderange_j(1, 63)
+sesse.data_partbuild.noderange_k(1, 1)
+sesse.data_partbuild.nodestep(1, 1, 1)
+sesse.data_partbuild.nodedelta(0, 0, 0)
+sesse.data_partbuild.description("Shuttle")
+sesse.data_partbuild.create()
+sesse.part.select_byname_begin("(CASE:Case 1)Shuttle")
+sesse.case.select("Case 1")
+sesse.data_partbuild.end()
+session.show("image", width=800, height=600)
+
```

Comment:
Could the load just use the shuttle basic load instead? Simplify the 20+ lines to load the data. Simply to not get distracted by 20+ lines of how to load structured data.

Reply:
@kecolburn Shuttle basic load did not have all the variables needed.